robot-server/Makefile

```
@@ -21,10 +21,10 @@ sdist_file = dist/$(call python_get_sdistname,robot-server,robot_server)
 # These variables can be overriden when make is invoked to customize the
 # behavior of pytest. For instance,
 # make test tests=tests/opentrons/tools/test_qc_scripts.py would run only the
-# specified test
+# specified test 
 tests ?= tests
 cov_opts ?= --cov=$(SRC_PATH) --cov-report term-missing:skip-covered --cov-report xml:coverage.xml
-test_opts ?=
+test_opts ?= -n auto --dist loadscope
```

Nitpick: Can we spell out full-length command line options, for readability?

```suggestion
test_opts ?= --numprocesses auto --dist loadscope
```